styles/Home.module.scss

```diff
@@ -0,0 +1,63 @@
+.description {
```

Comment:
The style should be unique, remove any unused styles.

Reply:
done